api/python/cellxgene_census/tests/experimental/pp/test_hvg.py

```diff
@@ -15,7 +15,7 @@ def small_mem_context() -> soma.SOMATileDBContext:
     """used to keep memory usage smaller for GHA runners."""
     cfg = {
         "tiledb_config": {
-            "soma.init_buffer_bytes": 64 * 1024**2,
+            "soma.init_buffer_bytes": 32 * 1024**2,
```

Comment:
@bkmartinjr With pytorch unit test OOM failure seemingly fixed, I was next seeing mysterious "workflow cancelled" failures, which I'm assuming meant OOM (no other errors available). This change seems to have helped. OK with lower memory usage? Experimental runs times may be a couple minutes slower, but hard to determine when comparing to past runs due to high variability in run times.